doc/book/src/installation.md

```diff
@@ -1,30 +1,34 @@
 # Installation
 
+## Requirements
+
+Maelstrom currently only supports Linux.
+
+...presumably there are also kernel version requirements?
+
+## Installed Binaries
+
 Maelstrom consists of the following binaries:
   - [`cargo-maelstrom`](cargo-maelstrom.md): A Rust test runner. This can be run in
     standalone mode &mdash; where tests will be executed on the local machine
     &mdash; or in clustered mode. In standalone mode, no other Maelstrom
     binaries need be installed, but in clustered mode, there must be a broker
     and some workers available on the network.
-  - [`maelstrom-broker`](broker.md): The Maelstrom cluster scheduler. There must be one of
+  - [`maelstrom-broker`](broker.md): The Maelstrom cluster broker. This component is
+    responsible for scheduling work onto nodes in the cluster. There must be one of
     these per Maelstrom cluster.
   - [`maelstrom-worker`](worker.md): The Maelstrom cluster worker. This must be
     installed on the machines in the cluster that will actually run jobs
     (i.e. tests).
   - [`maelstrom-run`](run.md): A Maelstrom client for running arbitrary commands on a
-    Maelstrom cluster. While this binary can run in standalone mode, it's only really
-    useful in clustered mode.
-
-The installation process is virtually identical for all binaries. We'll
-demonstrate how to install all the binaries in the following sections. You can
-pick and choose which ones you actually want to install.
```

Comment:
I didn't find these words added much, especially since the last sentence isn't true for `nix`.

Reply:
Yeah, that's all been nixed.